drivers/serial/uart_ene_kb1200.c

```diff
@@ -0,0 +1,380 @@
+/*
+ * Copyright (c) 2023 ENE Technology Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT ene_kb1200_uart
+
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/uart.h>
+#include <zephyr/drivers/pinctrl.h>
+#include <reg/ser.h>
+
+struct kb1200_uart_config {
+#ifdef CONFIG_UART_INTERRUPT_DRIVEN
+	void (*irq_cfg_func)(void);
+#endif
+	SER_T *ser;
```

Comment:
Can you add SER_T to https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/checkpatch/typedefsfile? Should clear this checkpatch error. I think.

Reply:
Can lower case type name fix this problem?

Reply:
I change "SER_T" to "struct serial_regs"

Reply:
thanks! yeah that's even better